runtime/src/iree-amd-aie/driver/xrt/native_executable.cc

```diff
@@ -267,8 +279,8 @@ static void iree_hal_xrt_native_executable_destroy(
 
   for (iree_host_size_t i = 0; i < executable->entry_point_count; ++i) {
     try {
-      delete executable->entry_points[i].kernel;
-      delete executable->entry_points[i].instr;
+      // delete executable->entry_points[i].kernel;
```

Comment:
Removable

Reply:
well no becaues i'm not sure if this works - @jtuyls added this when he was fighting the numerics/XRT issues. Currently I'm still figuring out if this passes on Linux (there's a new CI runner with an updated driver now).

Reply:
It passes on my machine but like I say: if it didn't happen on CI it didn't happen :)